<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connections</class>
 <widget class="QWidget" name="Connections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>508</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListView" name="list"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="driverLabel">
         <property name="text">
          <string>Driver:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="driver"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="hostnameLabel">
         <property name="text">
          <string>Host Name:</string>
         </property>
         <property name="buddy">
          <cstring>hostName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="hostName">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="databaseLabel">
         <property name="text">
          <string>Database Name:</string>
         </property>
         <property name="buddy">
          <cstring>database</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="database"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>User Name:</string>
         </property>
         <property name="buddy">
          <cstring>userName</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KLineEdit" name="userName"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="buddy">
          <cstring>password</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KLineEdit" name="password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="selectFile">
         <property name="text">
          <string>Open</string>
         </property>
         <property name="icon">
          <iconset theme="document-open"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="testConnection">
         <property name="text">
          <string>Test</string>
         </property>
         <property name="icon">
          <iconset theme="document-preview"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="testResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TestResult</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
